example/openMPSchedule/src/openMPSchedule.cpp

```diff
@@ -0,0 +1,163 @@
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED “AS IS” AND ISC DISCLAIMS ALL WARRANTIES WITH
+ * REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL ISC BE LIABLE FOR ANY
+ * SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR
+ * IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
+#    include <alpaka/alpaka.hpp>
+#    include <alpaka/example/ExampleDefaultAcc.hpp>
+
+#    include <cstdint>
+#    include <iostream>
+
+//#############################################################################
+//! OpenMP schedule demonstration kernel
+//!
+//! Demonstrates how to set OpenMP schedule and prints distribution of alpaka
+//! thread indices between OpenMP threads
+struct OpenMPScheduleKernel
+{
+    // A simple way to set a schedule for all invocations on this kernel using a constexpr member.
+    // Note that constexpr is not required, however otherwise there has to be an external definition.
+    // Another way to set the schedule is via trait specialization, as demonstrated after the kernel body.
+    // Trait specialization is checked first, so the member only has effect when the trait is not specialized for this
+    // type.
+    static constexpr auto ompSchedule = alpaka::omp::Schedule{alpaka::omp::Schedule::Static, 1};
+
+    //-----------------------------------------------------------------------------
+    template<typename TAcc>
+    ALPAKA_FN_ACC auto operator()(TAcc const& acc) const -> void
+    {
+        // For simplicity assume 1d index space throughout this example
+        using Idx = alpaka::Idx<TAcc>;
+        Idx const globalThreadIdx = alpaka::getIdx<alpaka::Grid, alpaka::Threads>(acc)[0];
+
+        // Print work distribution between threads for illustration
+        printf(
+            "alpaka global thread index %u is processed by OpenMP thread %d\n",
+            static_cast<std::uint32_t>(globalThreadIdx),
+            omp_get_thread_num());
+    }
+};
+
+
+namespace alpaka
+{
+    namespace traits
+    {
+        //! Schedule trait specialization for our kernel.
+        //! When uncommented, that would have priority over the ompSchedule member.
+        //! This is the most general way to define a schedule, while that member is a shortcut for the common case.
+        //! In case neither the trait nor the member are provided, alpaka does not set any runtime schedule and the
+        //! schedule used is defined by omp_set_schedule() called on the user side, or otherwise by the OMP_SCHEDULE
+        //! environment variable.
+        /*
+        template<typename TAcc>
+        struct OmpSchedule<OpenMPScheduleKernel, TAcc>
```

Comment:
Could we provide this as a second example or the second file in this example. 
This code will be soon outdated and untested.

Reply:
Good point, will change